### ISSUE TITLE
Design a logo

### ISSUE THREAD

**Issue by breandan** [acejump/AceJump#313]
IdeaVim has a [spiffy new logo](https://plugins.jetbrains.com/files/164/72656/icon/META-INF_pluginIcon.svg). It would be nice to have a logo for AceJump as well.

**Comment by fools-mate** [comment]
Here's a quick idea. What do you think?

![image](https://user-images.githubusercontent.com/34334994/81557732-c5b68880-938c-11ea-87c9-183b26f2df68.png)

**Comment by breandan** [comment]
Very creative! I like the resemblance between form and function! Do you have any ideas for a [square logo](https://www.jetbrains.org/intellij/sdk/docs/basics/plugin_structure/plugin_icon_file.html#plugin-logo-shape)? I wonder if there is a way to incorporate the jump motion, or depict the tag somehow.

![image](https://user-images.githubusercontent.com/175716/81585743-913dcf00-9382-11ea-9a9f-6465f93f746f.png)

Another direction would be to use a kind of aviation themed logo, with `⚞[A]⚟` in wings:

![image](https://user-images.githubusercontent.com/175716/81585015-89c9f600-9381-11ea-8950-2b7e6b0e03ac.png) ![image](https://user-images.githubusercontent.com/175716/81585047-93ebf480-9381-11ea-8dc8-9f3feecf5de8.png) 
![image](https://user-images.githubusercontent.com/175716/81585128-b4b44a00-9381-11ea-80cd-022c17ec3463.png) ![image](https://user-images.githubusercontent.com/175716/81585196-d01f5500-9381-11ea-8e5f-f60e85762f83.png)

**Comment by fools-mate** [comment]
Thank you for your kind feedback.

I forgot that the plugin requires a square version. I tried to put only the "jumping ace" in a square, but it looks a bit lost. In the next days I will try to develop a new version. References are always very helpful and I also thank you for the guidelines, I will follow them.

**Comment by fools-mate** [comment]
Here are two new versions, I hope you like them. 

The aviation theme would work in theory, but since there is "jump" in the name, I think we should stick to this.

![Logo V3](https://user-images.githubusercontent.com/34334994/81683375-63c25580-9456-11ea-81bf-e629a136094a.jpg)
![Logo V2](https://user-images.githubusercontent.com/34334994/81683378-64f38280-9456-11ea-9d8f-8a1f33d6f5ab.jpg)

**Comment by breandan** [comment]
Looks good! I like how it captures the cursor motion, but wonder if there was a way to make it more recognizable at a glance to users who are familiar with the “AceJump” name. What about something like this?

![EFBF43A6-4177-429F-9076-8175A0C4A67B](https://user-images.githubusercontent.com/175716/81732766-f36c0200-945e-11ea-8bc2-3a1bbda6b321.jpeg)
![81D0E72F-55FD-4E5F-B644-5B33EBC020CA](https://user-images.githubusercontent.com/175716/81732792-fbc43d00-945e-11ea-98a7-7fc1aff657e9.jpeg)



Or if you like the playing card theme, maybe the shape of the cursor could become a playing card like so:

![A5937A7B-441F-484E-BD08-7F131F7755A4](https://user-images.githubusercontent.com/175716/81732734-e7804000-945e-11ea-9f5b-ac02fbdf0c47.jpeg)

**Comment by breandan** [comment]
@johnlindquist Any thoughts?

**Comment by AlexPl292** [comment]
Sorry for the interruption, just wanted to say that I personally like the above versions with square logo from @fools-mate and @breandan's first sketch :)

**Comment by johnlindquist** [comment]
@breandan i like the highlighted A in the first one a lot. It's a cool callout to how you actually use it.

**Comment by breandan** [comment]
That's what I was thinking, I love how @fools-mate's original design shows off the usage. If only there was a way to make it square and give it some kind of "action", maybe with an exclamation point:

```
🄰ce
Jump!
```

**Comment by fools-mate** [comment]
@breandan You have to keep in mind that the icon is normally only displayed in the size 40x40px. Only once on the plugin marketplace it would be 80x80px, but that's not much either. I really like your first sketch/idea, but the text and the details will be too small. So I tried to find a unique and recognizable symbol that could stand for itself.

Maybe we just take the highlighted A for the marketplace and for Github and so on I implement your idea?

**Comment by breandan** [comment]
Good point, I was not thinking about the pixel constraints. Maybe to align with the platform branding somewhat, we could keep it minimal and just use `AJ` with the highlighted A on a dark background: 

![image](https://user-images.githubusercontent.com/175716/81778945-b9801780-94c1-11ea-9191-c3fb1c6e3b52.png)

Also happy to use the previous design... now that I look at it more closely I can see the A in the spade and the lower case `j` tilted in the jump.

**Comment by fools-mate** [comment]
Here a quick size reference:

![size-reference](https://user-images.githubusercontent.com/34334994/81781501-7d1bde00-94f8-11ea-8ffe-7b3654a26712.jpg)

**Comment by fools-mate** [comment]
I want everyone to be happy with the end result. We can still try a few things, I'm always open to it.

I'm glad you saw that little detail with the A and the jumping J. I thought it was a clever idea. :)

**Comment by breandan** [comment]
Yeah, I guess it does look a bit crowded.

![](https://user-images.githubusercontent.com/34334994/81683378-64f38280-9456-11ea-9d8f-8a1f33d6f5ab.jpg)

Looking at V3 again I can also see the nib of a fountain pen, which is kinda evocative of the writing theme.

![](https://user-images.githubusercontent.com/34334994/81781501-7d1bde00-94f8-11ea-8ffe-7b3654a26712.jpg)

So far, we have the following pros and cons:

**V2**: Ace of spades, jumping j, fountain pen. A bit subtle.

**V1/V3**: Familiar, explanatory, but hard to read at lower resolution.

**V4**: Highlighted`A`. Recognizable, if you already use AceJump.

**Comment by breandan** [comment]
I think we have enough here to make a decision. Unless anyone else wants to throw in an idea, let’s sleep on it and circle back next week. Great work, @fools-mate!

**Comment by fools-mate** [comment]
All right, I'm curious what the next step is.  I would like to talk about colors again when there is a closer selection.

**Comment by fools-mate** [comment]
Hi @breandan, It became very quiet here. Any updates?

**Comment by breandan** [comment]
Here's a [Twitter poll](https://twitter.com/breandan/status/1274025876024438784), we should have the results in 7 days. If you can submit a PR containing the winning SVG we'll give you credit in the release notes and GitHub page. Thanks to @fools-mate and everyone for their design feedback!

**Comment by breandan** [comment]
Looks like the original design won out. Since it was almost tied, let's use the second one? If you can you submit a PR containing the SVG as a file named `pluginIcon.svg` to [this directory](https://github.com/acejump/AceJump/tree/master/src/main/resources/META-INF), I'll make sure to add your name to the contributors. Thanks again!

**Comment by fools-mate** [comment]
As I have already mentioned, I wanted to change the colours. So I did. I colored the "jump" part more energetic and tried to move the look more towards the Jetbrains style. I hope you like it?

I can do a PR then, no problem.

![pluginIcon@3x](https://user-images.githubusercontent.com/34334994/90337738-c04c8800-dfe4-11ea-87a6-47eda07f6709.png)

**Comment by breandan** [comment]
Looks really sharp, I think you nailed it. Thanks so much for your creativity and attention to detail.
